tensorflow/lite/c/common.c

```diff
@@ -119,7 +119,8 @@ void TfLiteSparsityFree(TfLiteSparsity* sparsity) {
   }
 
   if (sparsity->dim_metadata) {
-    for (int i = 0; i < sparsity->dim_metadata_size; i++) {
+    int i;
```

Comment:
Can you declare and assign `i`?

int i = 0;
for (; i < sparsity->dim_metadata_size; i++) {

Reply:
Applied